website_sale_product_attribute_filter_category/migrations/15.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env["ir.ui.view"].search(
+        [
+            (
+                "key",
+                "=",
+                "website_sale_product_attribute_filter_category"
+                ".products_attributes_collapsible",
+            )
+        ]
+    ).unlink()
```

Comment:
During upgrade we try to minimize ORM operations:
```suggestion
    openupgrade.logged_query(
        env.cr,
        """
        DELETE FROM ir_ui_view
        WHERE key = 'website_sale_product_attribute_filter_category.products_attributes_collapsible'
        AND website_id IS NOT NULL
        """,
    )
```

Reply:
changes made! thanks for the suggestion :)